corehq/apps/export/custom_export_helpers.py

```diff
@@ -507,12 +507,19 @@ def update_multi_select_column(col):
         dont_show_cols = {"sync_log_id"}
         for table in table_conf:
             if table.get("index", "") == "#.actions.#":
-                for col in table.get("column_configuration", []):
+                column_configuration = table.get("column_configuration", [])
+                for col in column_configuration:
                     index = col.get("index", "")
                     if index not in dont_show_cols:
                         col["show"] = True
                     else:
                         dont_show_cols.discard(index)
+
+                    # Remove attachements - This is specfically in response to
+                    # http://manage.dimagi.com/default.asp?160461#891283
```

Comment:
(super-nitpicky/not necessary to change) i don't like leaving the random fogbugz event anchors in urls